code/modules/mob/living/simple_animal/abnormality/zayin/bottle.dm

```diff
@@ -51,7 +57,41 @@
 		if(cake > 0)
 			user.adjustBruteLoss(-500) // It heals you to full if you eat it
 			icon_state = "bottle2"	//cake looks eaten
+
+	if(work_type == "Drink")
+		user.apply_status_effect(STATUS_EFFECT_TEARS)
+		to_chat(user, "<span class='userdanger'>You feel your strength sapping away...</span>")
+
 	return ..()
 
 
+/datum/status_effect/tears
```

Comment:
Might want to add a visible status effect icon for it, and a notification on both removal and application, just so people know what exactly happened.